src/styles/global.css

```diff
@@ -13,4 +13,13 @@ body {
   padding: 0;
   margin: 0;
   box-sizing: border-box;
+}
+
+.main-container {
```

Comment:
no need to do it in this PR, but I think it would be nice to have a responsive main-container, which would adjust based on media queries. I would check short tutorial or stackoverflow for how you can easily do this, from the top of my head with pure css you can use media queries. My favourite is to use flexbox, it's pure css and once you understand the mechanics it is absolutely great! (most css libraries use flex under the wood to create responsive designs). This short guide is quite comprehensive :) https://css-tricks.com/snippets/css/a-guide-to-flexbox/

Reply:
I will follow up on this in our call ahead.